core/src/main/java/com/novoda/noplayer/subtitles/NoSubtitlesStyle.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.noplayer.subtitles;
+
+import android.graphics.Typeface;
+
+import androidx.annotation.Nullable;
+
+class NoSubtitlesStyle implements SubtitlesStyle {
```

Comment:
Is it a `NoSubtitlesStyle`? `FallbackOnlySubtitlesStyle`?

Reply:
Yeah, I wasn't sure as the name is a bit ambiguous as it could refer to the style when there is no subtitles. They are not all fallback as text size just multiplies by 1. 
I'll rename it to `NoOpSubtitlesStyle`

Reply:
https://github.com/novoda/no-player/pull/277/commits/5c139883cb015dcbf3a89eb24283d48e5fa52713